README.md

```diff
@@ -55,7 +55,7 @@ Your skills contribution pull requests must include the following:
 
 Taxonomy skill files must be a valid [YAML](https://yaml.org/) file named `qna.yaml`. Each `qna.yaml` files contains a set of key/value entries with the following keys:
 
-- `task_description`: A description of the skill. **Required**
+- `task_description`: A description of the skill. This information is very critical and should be accurate. You should be able to mention `precise` to get precise generated data **Required**
```

Comment:
```suggestion
- `task_description`: A description of the skill.
This description is included in the prompt to the teacher model for generating the synthetic data.
So you should make sure to use an accurate and sufficiently detailed description to guide the teacher model.
The description can include instructions such as being precise or using brief answers. **Required**
```

Reply:
@skairali Can you also look to make sure that task_description under knowledge is also updated to match this?